test/integration/framework/test_server.go

```diff
@@ -128,7 +128,7 @@ func StartTestServer(t *testing.T, stopCh <-chan struct{}, setup TestServerSetup
 	}
 	go func() {
 		if err := kubeAPIServer.GenericAPIServer.PrepareRun().Run(stopCh); err != nil {
-			t.Fatal(err)
+			t.Error(err)
```

Comment:
what will cover causing a `FailNow` if this errors?

Reply:
Error is basically a Log followed by Fail right?

Reply:
`Fail` is different than `FailNow`

Reply:
"Calling FailNow does not stop those other goroutines." So, does `Fatal` here achieve the expected behavior? I'm somewhat confused myself.

Reply:
ok I agree, the existing Fatal call is relying on undefined behavior at best, let's see what happens

Reply:
It seems somebody has already made another PR on this which was accepted so removing this from current PR.